### DISCUSSION TITLE
How to handle (import??) custom linting rules.

### DISCUSSION THREAD

**Discussion by jwbanning** [postman-open-technologies/lifecycle#106] [Ideas]
When linting and custom rules are supported it would be great to have a blue print about moving those over to Postman

**Comment by kinlane** [comment]
Great suggestion. I'd like to pull together a plan for where this should occur, as a custom rule, or as a test script -- I have a blueprint started to consider how governance fits in with the existing Postman contract, performance, and security testing. I will ping you when I have more thoughts here, or will pick your brain.
